src/test/rgw/test_rgw_lua.cc

```diff
@@ -62,11 +62,11 @@ class TestUser : public sal::User {
     return 0;
   }
 
-  virtual int store_info(const DoutPrefixProvider *dpp, optional_yield y, const RGWUserCtl::PutParams& params) override {
+  virtual int store_by_id(const DoutPrefixProvider* dpp, optional_yield y, bool exclusive, RGWUserInfo* old_info) override {
```

Comment:
what does 'exclusive' signify here? 
old_info is the old RGWUserInfo to be returned by backend store right?

Reply:
Exclusive means what it means to RGWUserCtl::store_info().  Not a good answer, I know.  Tracing it through ~15 layers of indirection (!), it appears that it means "exclusive create".  It presumably does nothing if it's not a create.

You are correct for old_info.